server/graphql/v2/object/Comment.js

```diff
@@ -1,5 +1,6 @@
-import { GraphQLObjectType, GraphQLString } from 'graphql';
+import { GraphQLList, GraphQLObjectType, GraphQLString } from 'graphql';
 import { GraphQLDateTime } from 'graphql-iso-date';
+import { GraphQLJSONObject } from 'graphql-type-json';
```

Comment:
For consistency, use `import GraphQLJSON from 'graphql-type-json';`.

Reply:
@kewitz : Done. :+1: